webpack.mix.js

```diff
@@ -16,4 +16,9 @@ mix.scripts([
     './node_modules/feather-icons/dist/feather.min.js',
     './node_modules/eclipsefdn-solstice-assets/js/solstice.cookies.js',
     './node_modules/eclipsefdn-solstice-assets/js/solstice.js'
-], './static/js/solstice.js');
\ No newline at end of file
+], './static/js/solstice.js');
+
+
+mix.scripts([
+    './node_modules/eclipsefdn-solstice-assets/js/solstice.tables.js'
+], './static/js/hugo-custom.js');
```

Comment:
how about we call this file solstice.hugo.js?

Reply:
Done